eduHub/course/src/main/kotlin/com/arsiu/eduhub/course/application/mapper/CourseMapper.kt

```diff
@@ -44,3 +46,26 @@ interface CourseMapper {
     fun toDtoResponseList(courses: List<Course>): List<CourseDtoResponse>
 
 }
+
+@Mapper(componentModel = "spring")
+interface CourseToEntityMapper {
```

Comment:
Your application layer should know nothing about infrastructure layer. 

Mongo Entity is a part of infrastructure layer

Reply:
`+`